docs/content/users/extend/customization-extendibility.md

```diff
@@ -111,28 +111,39 @@ web_extra_exposed_ports:
 
 ## Providing Custom Environment Variables to a Container
 
-Custom environment variables may be set in
+You can set custom environment variables in several places:
 
-* The project `.ddev/.env file`, which could look something like this:
+* The project’s [`web_environment`](../configuration/config.md#web_environment) setting in `.ddev/config.yaml`:
```

Comment:
```suggestion
* The project’s [`web_environment`](../configuration/config.md#web_environment) setting in `.ddev/config.yaml or `.ddev/config.*.yaml`:
```

Reply:
Ha, I almost got rid of that YAML reference to let the config page explain and make this easier to read, but kept it because the location seemed urgently important in this case. Might as well call this out for technical accuracy!